sdk/python/tests/dsl/component_bridge_tests.py

```diff
@@ -0,0 +1,137 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import unittest
+import kfp
+from kfp.components import load_component_from_text
+from kfp.dsl.types import InconsistentTypeException
+
+
+class TestComponentBridge(unittest.TestCase):
+    # Alternatively, we could use kfp.dsl.Pipleine().__enter__ and __exit__
+    def setUp(self):
+        self.old_container_task_constructor = kfp.components._components._container_task_constructor
+        kfp.components._components._container_task_constructor = kfp.components._dsl_bridge._create_container_op_from_component_and_arguments
+
+    def tearDown(self):
+        kfp.components._components._container_task_constructor = self.old_container_task_constructor
+
+    def test_conversion_to_container_op(self):
+        component_text = '''\
```

Comment:
Shall we use `textwrap.dedent` to make this prettier?

Reply:
Thanks for the suggestion.
Done.